### DISCUSSION TITLE
An idea for accessible graphics

### DISCUSSION THREAD

**Discussion by MiguelUB** [vega/altair#3314] [Show and tell]
Hello!

I'd like to share my personal project with you a library that complements Altair to make chart creation more accessible. My approach involves adding accessible themes with models for easy theme creation. On the other hand, there's a function that generates an HTML to render the chart in SVG, with options for customization.

Here are the links:
Repository -> https://github.com/MiguelUB/altair-easeviz
Documentation -> https://miguelub.github.io/altair-easeviz/
Example website(in spanish) ->https://accesible-theme-altair.onrender.com

Hope your opinion!

**Comment by mattijn** [comment]
Thanks for sharing! Really interesting approach. Maybe we can use concepts of your library to solve the following issues as well:
- https://github.com/altair-viz/altair/issues/2820
- https://github.com/altair-viz/altair/issues/2103

cc: @joelostblom

**Comment by joelostblom** [comment]
Cool! Thank you for sharing @MiguelUB. I really like that there is an option to control all font sizes with a single function. I think that is convenient and as @mattijn said, if there is something that is easy portable to altair to solve #2820, I would be happy to chat more about it

**Comment by MireiaUB** [comment]
Hi,
I've been reviewing this project and it was not featured anywhere. Accessibility is an increasing concern and I think this could be a first step.

**Reply by mattijn** [reply]
Agree! Linked it to another issue about documentation on best practices, so that this work is considered once we (as in the community, which can be anyone) gets to it.
Thanks for bringing this back on the table!

**Comment by MireiaUB** [comment]
[heart]         Mireia Isabel Ribera Turro reacted to your message:
________________________________
